frontend/src/components/Analysis/utils.tsx

```diff
@@ -163,35 +189,49 @@ export function parseFineGrainedResults(
       const myAnalysis = system.system_info.analyses[analysisIdx];
       const myResult = singleAnalysis.analysis_results[analysisIdx];
 
-      // Skip non-bucketing analyses for now
-      if (myResult.cls_name !== "BucketAnalysisResult") {
-        continue;
-      }
       const analysisName = myResult.name;
-      const analysisBuckets: BucketPerformance[] =
-        myResult["bucket_performances"];
       const analysisDescription = myAnalysis.description;
       const bucketType = myAnalysis["method"];
 
-      const metricToParsed: { [metric: string]: ResultFineGrainedParsed } =
-        parse(
-          analysisBuckets,
+      if (myResult.cls_name === "BucketAnalysisResult") {
+        const metricToParsed = parseBucketAnalysisFeatures(
+          myResult["bucket_performances"],
           bucketType,
           myResult.level,
           analysisName,
           analysisDescription
         );
-      for (const [metric, singleResult] of Object.entries(metricToParsed)) {
-        if (!(metric in parsedResults)) {
-          parsedResults[metric] = {};
-        }
-        if (!(analysisName in parsedResults[metric])) {
-          parsedResults[metric][analysisName] = [];
+
+        for (const [metric, singleResult] of Object.entries(metricToParsed)) {
+          if (!(metric in parsedResults)) {
+            parsedResults[metric] = {};
+          }
+          if (!(analysisName in parsedResults[metric])) {
+            parsedResults[metric][analysisName] = [];
+          }
+          parsedResults[metric][analysisName].push(singleResult);
         }
-        parsedResults[metric][analysisName].push(singleResult);
+      } else if (myResult.cls_name === "ComboCountAnalysisResult") {
+        const parsedComboAnalysis = parseComboCountFeatures(
+          myResult["combo_counts"],
+          myResult.level,
+          analysisName,
+          analysisDescription
+        );
+        parsedComboAnalyses.push(parsedComboAnalysis);
+      } else {
```

Comment:
This else statement doesn't seem to be necessary.

Reply:
Oh this was just a hint to let future developers know where to change to support new analyses. I'll remove it then.